apps/librelingo_yaml_loader/librelingo_yaml_loader/yaml_loader.py

```diff
@@ -40,8 +36,20 @@ def add_bool(self, node):
 
 def _load_yaml(path: Path):
     """Helper function for reading a YAML file"""
-    with open(path, encoding="utf-8") as yaml_file:
-        return load(yaml_file, Loader=SafeLoader)
+    with open(path) as yaml_file:
+        try:
+            from yaml import CSafeLoader
+
+            return load(yaml_file, Loader=CSafeLoader)
+        except ImportError:
+            print(
```

Comment:
Well, it seems like this is printing it to standard output, which is not the ideal place to show this warning in, as warnings/debug messages are not part of the intended output

Reply:
perhaps you should use the `logging` module for this? https://stackoverflow.com/a/6579522